rfc/rfc006-distributed-registry.md

```diff
@@ -64,7 +64,9 @@ base58char = "1" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" / "A" / "B" / "
 
 The `idstring` is derived from the public part of a key pair that was used to sign the transaction that created the DID document.
 
-`idstring = BASE-58(SHA-256(raw-public-key-bytes))`
+`idstring = BASE-58(hash)`
+
+The `hash` is calculated conforming [rfc7638](https://tools.ietf.org/html/rfc7638), using the `SHA256` hashing algorithm.
```

Comment:
conform to, drop the comma

Do we need to specify the base58 alphabet?

Reply:
it already is at line 57....